core/pom.xml

```diff
@@ -468,6 +468,11 @@ THE SOFTWARE.
       <artifactId>commons-logging</artifactId>
     </dependency>
     <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
These properties are for maven plug-ins, not for libraries. Most properties of dependencies are hard-coded. (But anyway, I also think that properties are easier to change. My point is that we should then do this for *all* and not just for a subset of these).